SPECS/python-kubernetes/python-kubernetes.spec

```diff
@@ -1,17 +1,15 @@
 %global library kubernetes
-%global basehash d30f1e6fd4e2725aae04fa2f4982a4cfec7c682b
+%global basehash 06191d321491abd87caaf0d049030dc87720684a
 %{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
```

Comment:
Nit: This line can be removed, since we define the macro in `mariner-rpm-macros`